### ISSUE TITLE
Lists and maps

### ISSUE THREAD

**Issue by jamtur01** [opencredo/terrahelp#8]
```
ERROR occured : Error occured decoding Terraform vars file: At 11:18: root.public_subnets: unknown type for string *ast.ListType
```

Any plans to update to support list and map variables?

Thanks!

**Comment by nickithewatt** [comment]
@jamtur01 can certainly look into that!

**Comment by nickithewatt** [comment]
@jamtur01 this has now been done and merged, available in 0.4.2

**Comment by jamtur01** [comment]
Awesome - thanks so much!
